### PR TITLE
Fix release script and packaged app errors

### PR DESCRIPTION
**Pull request by ewc340**

### CODE REVIEW

**Comment by ewc340** [comment]
Still need to test this a bit more so won't merge this yet

**Review by HaileyJang — COMMENTED** [review]
good

**Review by HaileyJang — APPROVED** [review]
good

File: package.json
Comment (ewc340) [inline package.json:111]: These are basically devDependencies and don't need to be dependencies for the actual app, this way we save space for the size of the packaged app.

**Comment by ewc340** [comment]
This PR should be good to go after we fix the UID display issue. After that we should also finally merge to master.

File: package.json
Comment (ewc340) [inline package.json:102]: These dependencies allow commands `mkdir -p` and `rm -rf` to be cross-platform (Windows, Mac, Linux).

**Review by AVSurfer123 — APPROVED** [review]

**Pull request merged**
